pkg/config/config_template.yaml

```diff
@@ -385,14 +385,17 @@ api_key:
   # max_events_per_second: 200
 
   ## @param max_memory - integer - optional - default: 500000000
-  ## Maximum memory to allow for the trace Agent. The Agent is killed if this number is surpassed.
+  ## This value is what the agent will aim to use in terms of memory. If surpassed, the API
```

Comment:
/cc @l0k0ms made some changes here to reflect new behavior, and also reworded a bit. PTAL